Makefile.env

```diff
@@ -16,3 +16,10 @@ GOBIN=$(shell go env GOBIN)
 endif
 
 GOPATH=$(shell go env GOPATH)
+
+# If true, run tests using -v option (verbosity)
+export TEST_VERBOSITY ?= 0
+
```

Comment:
I suggest defining the `TEST_OPTIONS` as a separate variable, we even can get it for environment, so it will allow as to provide additional arguments to the tests. For example, `-timeout` or `-json`

Reply:
ok